test/test_pwm_setup.py

```diff
@@ -13,8 +13,8 @@ def test_start_pwm(self):
         pwm_test = '/sys/class/pwm/pwmchip0/pwm0'
 
         assert os.path.exists(pwm_test)
-        duty = open(pwm_test + '/duty_cycle').read()
-        period = open(pwm_test + '/period').read()
+        duty = open(pwm_test + '/duty_cycle').readline().strip()
```

Comment:
Yeah, every string assert was failing because of the return line.  Still get 2 failures, but it's better than the whole test case failing.